docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -89,16 +87,16 @@ function yukonca_glider_preprocess_paragraph(&$variables) {
       $start_time = explode(".", $paragraph->field_opening_time->value / (60 * 60));
       $end_time = explode(".", $paragraph->field_closing_time->value / (60 * 60));
       if (isset($start_time[1])) {
-        $variables['start_time'] = $start_time[0] . ' h ' . date("i", $paragraph->field_opening_time->value);
+        $variables['start_time'] = $start_time[0] . ' h 30';
```

Comment:
It looks like this is forcing "h 30" if there is a minute value. Is that right? What's going on here?